client/src/Components/Common/CommentContainer/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { List } from 'antd';
+import PropTypes from 'prop-types';
+import Comment from '../Comment';
+import MainButton from '../MainButton';
+
+const commentsPerPage = 5;
+let arrayForHoldingComments = [];
+
+const CommentContainer = ({ data }) => {
+  const [commentsToShow, setCommentsToShow] = useState([]);
+  const [next, setNext] = useState(5);
+
+  const loopWithSlice = (start, end) => {
+    const slicedComments = data.slice(start, end);
+    arrayForHoldingComments = [...arrayForHoldingComments, ...slicedComments];
+    setCommentsToShow(arrayForHoldingComments);
+  };
+
+  const handleShowMoreComments = () => {
+    loopWithSlice(next, next + commentsPerPage);
+    setNext(next + commentsPerPage);
+  };
```

Comment:
nice work here by saving the old comments and add the new ones to them but you can use less code to achieve that since you are using the component state, by that you don't need the variable ```arrayForHoldingComments``` to store the data in it instead you should use the component state
```suggestion
const commentsPerPage = 5;

const CommentContainer = ({ data }) => {
  const [commentsToShow, setCommentsToShow] = useState([]);
  const [next, setNext] = useState(5);

  const loopWithSlice = (start, end) => {
    const slicedComments = data.slice(start, end);
    setCommentsToShow([...commentsToShow, ...slicedComments]);
  };

  const handleShowMoreComments = () => {
    loopWithSlice(next, next + commentsPerPage);
    setNext(next + commentsPerPage);
  };

```